frontend/src/metabase/dashboard/components/DashCard/DashCardMenu/DashCardMenuItems.tsx

```diff
@@ -0,0 +1,101 @@
+import { useMemo } from "react";
+import { t } from "ttag";
+
+import type { DashCardCustomMenuItem } from "embedding-sdk";
+import { useInteractiveDashboardContext } from "embedding-sdk/components/public/InteractiveDashboard/context";
+import { editQuestion } from "metabase/dashboard/actions";
+import type { DashCardMenuItem } from "metabase/dashboard/components/DashCard/DashCardMenu/DashCardMenu";
+import { useDispatch } from "metabase/lib/redux";
+import { Icon, Menu } from "metabase/ui";
+import type Question from "metabase-lib/v1/Question";
+import type { Dataset } from "metabase-types/api";
+
+import { canDownloadResults, canEditQuestion } from "./utils";
+
+export const DashCardMenuItems = ({
+  question,
+  result,
+  loading,
```

Comment:
I think this prop could be misleading since it's a loading state from `useDownloadData` hook, and at glace, I confused this with the loading of the component itself (if any).

It means we're loading the results after clicking download button. I think we could rename this to something else e.g.

```suggestion
  isDownloadingData,
```